vck-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/helper/PresentationFactory.kt

```diff
@@ -65,6 +88,63 @@ internal class PresentationFactory(
         }
     }
 
+    /**
+     * Performs calculation of the [SessionTranscript] and [DeviceAuthentication],
+     * acc. to ISO/IEC 18013-5:2021 and ISO/IEC 18013-7:2024, if required in [request] (i.e. it will be encrypted)
+     */
+    @Throws(PresentationException::class, CancellationException::class)
+    private suspend fun calcDeviceSignature(
+        responseWillBeEncrypted: Boolean,
```

Comment:
rename to `encryptResponse`?

Reply:
We're in the step before actually encrypting the response, and this function also does not encrypt it, so I came up with this name -- can we leave it at that?